src/SampleWebApp/Views/AuthorsMvc/Delete.cshtml

```diff
@@ -0,0 +1,34 @@
+@model SampleWebApp.Models.Author
+
+@{
+    ViewData["Title"] = "Delete";
+}
+
+<h2>Delete</h2>
+
+<h3>Are you sure you want to delete this?</h3>
+<div>
+    <h4>Author</h4>
+    <hr />
+    <dl class="dl-horizontal">
```

Comment:
We don't use `<dl>` for form layout in the templates anymore. Please make it match https://github.com/aspnet/Templates/blob/dev/src/Rules/StarterWeb/IndividualAuth/Views/Account/Login.cshtml